features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_CORDIO/TARGET_NRF5x/stack/sources/pal_timer.c

```diff
@@ -25,9 +25,9 @@
  * Notes:
  *
  *    This is timer driver dedicated to scheduler, an interrupt will be triggered to do scheduler task
- *    when timer hits compare value. Timer1 is used here.
+ *    when timer hits compare value. TIMER2 is used here.
```

Comment:
can we ping the cordio team about this ? It would be nice if the timer used was selectable at compile time. Also, IIRC correctly we cannot use TIMER0 because it is already used by the stack. is that correct ? We should notice the HAL team that TIMER2 is reserved; this is new.